components/file-input/file-input.js

```diff
@@ -0,0 +1,38 @@
+const fileInputTemplate = `
+  <div class="pages-content-container">
+    <div class="pages-content-element">
+      <p class="lead">2. Choose the content type </p>
+      <p class="paragraph">
+        Select supported content type from the list below and upload the file from your computer.
+      </p>
+      <select class="dropdown" name="content-type">
+        <option value="">--Please select an option--</option>
+        <option value="3D">3D Object (.gltf, .glb; max size 50MB)</option>
+        <option value="Image">Image (.jpg, .png, .gif; max size 15MB)</option>
+        <option value="Audio">Audio (.mp3; max size 10MB)</option>
+        <option value="Video">Video (.mp4; max size 25MB)</option>
+      </select>
+      <a class="passive-button">Upload file</a>
+    </div>
+    <div class="pages-content-element">
+      <div class="image-placeholder">
+        Click to upload
+      </div>
+    </div>
+  </div>`;
```

Comment:
How do we want to deal with styles in web components? @nicolocarpignoli

Reply:
This is how I did it due to the shadowDOM being in the way of CSS selectors. I think it works okay, but let me know if you find a better way.

https://github.com/AR-js-org/studio/pull/29/commits/cf270aedf1c00d815c318d610453e2401c11d70c

Reply:
for me it's ok, to start this way :)